.github/workflows/test.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        nodeversion: ['12.x', '13.x', '14.x', '15.x', '16.x', '17.x']
+        nodeversion: ['16.x', '17.x', '18.x', '19.x']
```

There's little point in supporting v17, which was never going to be LTS. Its end-of-life has already passed.
```suggestion
        nodeversion: ['16.x', '18.x', '19.x']
```